qiita_ware/test/test_ebi.py

```diff
@@ -398,6 +420,26 @@ def test_generate_demultiplexed_fastq(self):
         self.assertItemsEqual(obs_demux_samples,
                               ebi_submission.samples_prep.keys())
 
+    def test_generate_send_sequences_cmd(self):
+        ppd = self.write_demux_files(PrepTemplate(1))
+        e = EBISubmission(ppd.id, 'ADD')
+        e.generate_demultiplexed_fastq()
+        self.files_to_remove.append(e.ebi_dir)
+        e.write_xml_file(e.generate_study_xml(), e.study_xml_fp)
+        e.write_xml_file(e.generate_sample_xml(), e.sample_xml_fp)
+        e.write_xml_file(e.generate_experiment_xml(), e.experiment_xml_fp)
+        e.write_xml_file(e.generate_run_xml(), e.run_xml_fp)
+        e.write_xml_file(e.generate_submission_xml(), e.submission_xml_fp)
+        obs = e.generate_send_sequences_cmd()
+
+        exp = ('ascp --ignore-host-key -L- -d -QT -k2 '
```

Comment:
Here you should know which are the successful files (since you're able to test that elsewhere). The only reason for the existence of a test for such function is to test that the formatting is correct - but if you're executing the same formatting here then this is not testing anything...